tox.ini

```diff
@@ -13,7 +13,9 @@ deps =
     django19: Django >= 1.9,<1.10
     django110: Django >= 1.10,<1.11
     django111: Django >= 1.11,<1.12
+    django20: Django >= 2.0a1,<2.1
     tc: django-threadedcomments >= 1.1
     django-dev: https://github.com/django/django/tarball/master
+    mock
```

Comment:
mock is likely not needed?

Reply:
Really? I couldn't run `tox` without this. The error is following:

```
tox
GLOB sdist-make: /home/petr/soubory/programovani/Auto-mat/upstream/django-fluent-comments/setup.py
py27-django18 create: /home/petr/soubory/programovani/Auto-mat/upstream/django-fluent-comments/.tox/py27-django18
py27-django18 installdeps: Django >= 1.8,<1.9
py27-django18 inst: /home/petr/soubory/programovani/Auto-mat/upstream/django-fluent-comments/.tox/dist/django-fluent-comments-1.4.3.zip
py27-django18 installed: certifi==2017.11.5,chardet==3.0.4,Django==1.8.18,django-contrib-comments==1.8.0,django-crispy-forms==1.7.0,django-fluent-comments==1.4.3,django-tag-parser==3.1,idna==2.6,python-akismet==0.3.0,requests==2.18.4,urllib3==1.22
py27-django18 runtests: PYTHONHASHSEED='3456249481'
py27-django18 runtests: commands[0] | ./runtests.sh
Using Python version 2.7.1 from /home/petr/soubory/programovani/Auto-mat/upstream/django-fluent-comments/.tox/py27-django18/bin/python
Using Django version 1.8.18 from /home/petr/soubory/programovani/Auto-mat/upstream/django-fluent-comments/.tox/py27-django18/local/lib/python2.7/site-packages/django
Creating test database for alias 'default'...
.......E
======================================================================
ERROR: example.article.tests.test_moderation (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: example.article.tests.test_moderation
Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/home/petr/soubory/programovani/Auto-mat/upstream/django-fluent-comments/example/article/tests/test_moderation.py", line 10, in <module>
    from mock import patch
ImportError: No module named mock


----------------------------------------------------------------------
Ran 8 tests in 0.312s

FAILED (errors=1)
Destroying test database for alias 'default'...
ERROR: InvocationError: '/home/petr/soubory/programovani/Auto-mat/upstream/django-fluent-comments/runtests.sh'
py27-django19 create: /home/petr/soubory/programovani/Auto-mat/upstream/django-fluent-comments/.tox/py27-django19
```